.github/workflows/distribute.yml

```diff
@@ -0,0 +1,111 @@
+name: Distribute
+
+on:
+  push:
+    branches: [develop]
```

Comment:
I like the idea of having builds for each commit that the QA team can test (if they want to). In that way they can test specific commits and find regressions easily.
We could set up a `master` branch that reflects what is already deployed to the Play Store (and it can automatically make the deploys in the future).

This won't cause PRs to fail because it only runs when something is merged to develop, and external contributors are not allowed to push directly to develop.

That being said, I'm open to changing the workflow if this doesn't work or you don't feel comfortable with it

Reply:
that makes sense to me! yeah having a build for every commit is helpful for QA when tracking down issues, so we should do that, especially if there are no trade offs to be made.